zerver/views/invite.py

```diff
@@ -156,6 +157,40 @@ def revoke_multiuse_invite(
     return json_success(request)
 
 
+@require_realm_admin
+@has_request_variables
+def edit_multiuse_invite(
+    request: HttpRequest,
+    user_profile: UserProfile,
+    prereg_id: int,
+    stream_ids: List[int] = REQ(json_validator=check_list(check_int)),
+    invite_as: int = REQ(json_validator=check_int),
+) -> HttpResponse:
+    if not user_profile.can_invite_others_to_realm():
+        # Guest users case will not be handled here as it will
+        # be handled by the decorator above.
+        raise JsonableError(_("Insufficient permission"))
```

Comment:
I guess we do not need this since we anyways allow admins to create multi-use links always irrespective of this setting.

Reply:
@alya @timabbott I got one more question setting this, do we want to allow creating multi-use links when `Who can invite users to this organization` setting is set to `Nobody`. Currently we allow it.